src/app/routes/routes.ts

```diff
@@ -42,12 +42,26 @@ export const routes = [
             (m) => m.SearchResultModule
           ),
       },
+      {
+        path: 'vp',
+        loadChildren: () =>
```

Comment:
Was originally thinking I should add vp to auth guard children, but maybe not if it is being routed to that after the dashboard component (which is authguarded) already loads?

Reply:
No, there is no need as i explained above. It's better if this will work like enrol page -> check for component `request-claim`. To be more precisely you might copy paste this method: https://github.com/energywebfoundation/switchboard-dapp/blob/develop/src/app/routes/registration/request-claim/request-claim.component.ts#L155